PlaceRouteHierFlow/placer/SeqPair.cpp

```diff
@@ -184,6 +184,7 @@ void SeqPairEnumerator::Permute() {
   if (!IncrementSelected()) {
     if (_enumIndex.second >= _maxEnum - 1) {
       _enumIndex.second = 0;
+      _negEnumerator.ResetCnt();
```

Comment:
@srini229 Also here.

Reply:
This enumerator is for ordered sequence pair. the neq enumerator shouldn't be reset here.
I'm trying to see why the sequence 021 021 is not getting generated.